vendor/brave-ios/Ledger/BATBraveLedgerObserver.mm

```diff
@@ -0,0 +1,25 @@
+//
+//  BATBraveLedgerObserver.h
+//  brave/vendor/brave-ios:brave_rewards_ios_framework_arch_shared_library_sources
+//
+//  Created by Kyle Hickinson on 2019-07-09.
```

Comment:
Is this header correct? I think we should be using MPL license header

Reply:
Good catch!